components/experimental/src/duration/provider/digital.rs

```diff
@@ -0,0 +1,139 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// Provider structs must be stable
+#![allow(clippy::exhaustive_structs, clippy::exhaustive_enums)]
+
+//! Data provider struct definitions for this ICU4X component.
+//!
+//! Read more about data providers: [`icu_provider`]
+
+use displaydoc::Display;
+use icu_provider::prelude::*;
+
+#[icu_provider::data_struct(DigitalDurationDataV1Marker = "duration/digital@1")]
+#[derive(Debug, Copy, Clone, PartialEq)]
+#[cfg_attr(
```

Comment:
This should be something like this 

```rust
pub struct DigitalDurationDataV1 {
    pub separator: Cow<'data, str>,

    pub padding: u8,
}
```

Reply:
To clarify, `padding` here refers to hour padding? Do we pick up one separator and derive hm, hms, and ms representations like the following:
`separator: ",", padding = 2 => hm: "hh,mm", hms: "hh,mm,ss", ms: "mm,ss"`?